scylla-server/src/routes/location.routes.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import LocationController from '../controllers/location.controller';
+
+const nodeRouter = Router();
+
+nodeRouter.get('/locations', LocationController.getAllLocations);
```

Comment:
this should just be /, in index.ts you have to add this router to the server like i did with the node router and there you can say /locations